src/DynamoCoreWpf/ViewModels/Watch3D/HelixWatch3DViewModel.cs

```diff
@@ -1792,6 +1793,9 @@ private static MeshGeometry3D DrawTestMesh()
 
         internal void UpdateNearClipPlane()
         {
+
+            if (camera == null) return;
```

Comment:
ok...it is  always safe to check. but if camera is null, then UpdateNearClipPlane should never be called. especially on the tests. thats a different story anyway.,